hivemind/proto/p2pd.proto

```diff
@@ -147,8 +159,9 @@ message PSRequest {
   optional bytes data = 3;
 }
 
+
```

Comment:
```suggestion
```

Looks like the second empty line is added accidentally - let's remove it from diff :)